guidelines/1.4.10.md

```diff
@@ -105,7 +105,15 @@ override func traitCollectionDidChange(_ previousTraitCollection: UITraitCollect
 
 ## Guidance for Android
 
-This section needs more content. [Contribute via Github](https://github.com/theappbusiness/accessibility-guidelines/) or [email](mailto:jeanfrancois@theappbusiness.com).
+All Views will automatically allocate its content, as long as there's no `height` restriction. Using `android:layout_height="wrap_content"` is strongly recommended whenever possible.
+
+If you're using a `ConstraintLayout`, make sure it has `match_parent` on both **height** and **width** to fill the whole viewport. Make sure all its child Views are properly constraint to the edges.
```

Comment:
Minor: `constrained` instead of `constraint`? 🤔

Reply:
Changed!